resources/views/discord/new.blade.php

```diff
@@ -45,7 +45,7 @@
                                 </a>
                             @else
                                 <a href="#" style="text-decoration: none;">
-                                    <button class="btn btn-primary center-block" disabled><em class="fab fa-discord"></em> &thinsp; Discord Account {{ $_account->discord_id }} Linked</button>
+                                Currently registered with Discord ID {{ $_account->discord_user ? $_account->discord_user['username'].'#'.$_account->discord_user['discriminator'] : $_account->discord_id }}. <br/>
```

Comment:
```suggestion
                                <button class="btn btn-primary center-block" disabled><em class="fab fa-discord"></em>
                                        Currently registered with Discord ID {{ $_account->discord_user ? $_account->discord_user['username'].'#'.$_account->discord_user['discriminator'] : $_account->discord_id }}.
                                    </button>
```

Is the removal of the button intentional? The styling on the page looks a bit odd without it!

<img width="713" alt="2021-02-03 at 17 45 12" src="https://user-images.githubusercontent.com/7726792/106787459-a44aaa80-6647-11eb-9199-4f5fe9d56e25.png">

Reply:
Oh Vs code why do you do these things... (will fix, wasn't intentional)